xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -92,11 +92,17 @@ void CGUIWindowPVRGuide::GetContextButtons(int itemNumber, CContextButtons &butt
     CPVRTimerInfoTagPtr timer(epg->Timer());
     if (timer)
     {
+      if (timer->GetTimerScheduleId() != PVR_TIMER_NO_PARENT)
+        buttons.Add(CONTEXT_BUTTON_EDIT_TIMER_RULE, 19243); /* Edit timer rule */
```

Comment:
Is "timer rule" an existing naming convention? If not I think "edit parent timer" would be better (some constants speak of "parent" already).